front-end/src/LoggedInUser/FEED/UserProfileCards.jsx

```diff
@@ -7,16 +7,19 @@ import AllBuddies from "./AllBuddies";
 
 const UserProfileCards = ({ allUsers, bffs }) => {
   allUsers.map(user=> {
+    console.log("user profile ", user)
+
     return ('user first name' , user.first_name , user.age)})
-  
+
   
   return (
     <div className="cardHolder">
       {allUsers.map(user => {
         return (
           <div className="card">
             <img className="pic" src={user.pic} alt="profile picture" />
-            <div className="name-age"> {user.first_name}, {user.age}, {user.my_location}</div>
+            <div className="name-age"> {user.first_name}, {user.age}, {user.my_location} </div>
+        <div>{user.points ? <div>you are {user.points}% matched</div> : <div>you are not matched </div>}</div>
```

Comment:
Could we change "You are {user.points}% matched" to "{user.points}% match"?
Additionally, "You are not matched" to "0% match"

Reply:
changed.  Thanks!